src/project-tests/technical-docs-tests.js

```diff
@@ -93,11 +93,11 @@ export default function createTechnicalDocsPageTests() {
                     '"navbar" is not a <nav> element ');
             });
 
-            it('9. The first element within the navbar should be a <header> which contains text that describes the topic ' +
+            it('9. The navbar element should contain one <header> element which contains text that describes the topic ' +
                 'of the technical documentation.',
                 function() {
-                    FCC_Global.assert.strictEqual(document.getElementById('navbar').firstElementChild.nodeName, 'HEADER', 'The first ' +
-                        'element within #navbar should be a <header> ');
+                    FCC_Global.assert.strictEqual(document.getElementById('navbar').getElementsByTagName('header').length, 1, 'The navbar ' +
+                        'element should contain ONE <header> element.');
```

Comment:
@tchaffee if you're updating this PR anyway, I would not capitalize the word "one". I think it looks a little funky. Normal casing should be fine.